matrix/searching_in_sorted_matrix.py

```diff
@@ -2,8 +2,9 @@
 
 
 def search_in_a_sorted_matrix(
-        mat: List[list], m: int, n: int, key: int or float) -> None:
-    '''
+    mat: List[list], m: int, n: int, key: int or float
```

Comment:
This `int or float` thing does not really work, does it?!?

Reply:
it does not produce any error, but should I change to `float ` from `int or float`

Reply:
My sense is that it should be something like `Union[int, float]` and in the import section add, `from typing import Union` but please read https://docs.python.org/3/library/typing.html#typing.Union to be sure.